packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -34,9 +36,9 @@ function SelectionTool(props: Props) {
   } = props;
 
   const camera = useThree((state) => state.camera);
-  const { dataToWorld, worldToData } = useVisCanvasContext();
+  const { worldToData } = useVisCanvasContext();
 
-  const [startPoint, setStartPoint] = useState<Vector2>();
+  const [startEvt, setStartEvt] = useState<CanvasEvent<PointerEvent>>();
```

Comment:
This is the main change: I'm now keeping the entire start event. Relying on a single `selection` state was too confusing. It would have required making the points in the `selection` object optional, which introduced a whole lot of complexity.